notebooks/238-deeployd-if/README.md

```diff
@@ -17,6 +17,14 @@ The result of notebook work demonstrated on the image below:
 
 >**Note**: Please be aware that a machine with at least 32GB of RAM is necessary to run this example.
 
+The [second notebook](238-deep-floyd-if-optimize.ipynb) is about the optimization by 8-bit post-training quantization and weights compression and consists of the following steps:
+1. Compress weights of the converted OpenVINO text encoder from [notebook](238-deep-floyd-if.ipynb) with NNCF.
```

Comment:
let's change 'notebook' to 'first notebook' everywhere in this list and in 238-deep-floyd-if-optimize.ipynb

Reply:
done

Reply:
please also fix it in 238-deep-floyd-if-optimize.ipynb